### PR TITLE
Update VS Editor to latest NuGet packages

### PR DESCRIPTION
**Pull request by KirillOsenkov**
Turn off the checked in copy of VS Editor forked source, use NuGet packages instead.

### CODE REVIEW

File: version-checks [hunk 1]
```diff
@@ -17,7 +17,7 @@ DEP[0]=md-addins
 DEP_NAME[0]=MDADDINS
 DEP_PATH[0]=${top_srcdir}/../md-addins
 DEP_MODULE[0]=git@github.com:xamarin/md-addins.git
-DEP_NEEDED_VERSION[0]=54a807366be9c932383b45883229dc1f359c8298
+DEP_NEEDED_VERSION[0]=4916e521a430d095d9a86d17ee3b264102ffe369
```
Comment (Therzok) [inline version-checks]: Why the md-addins bump?

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.csproj [hunk 2]
```diff
@@ -9651,7 +9655,6 @@
     <None Include="MonoDevelop.Ide.Editor.Highlighting\VSCodeImport\objective-c\language-configuration.json" />
     <None Include="MonoDevelop.Ide.Editor.Highlighting\VSCodeImport\objective-c\package.json" />
   </ItemGroup>
-  <Import Project="..\Mono.TextEditor.Platform\Mono.TextEditor.Platform.Def.projitems" Label="Shared" Condition="Exists('..\Mono.TextEditor.Platform\Mono.TextEditor.Platform.Def.projitems')" />
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.csproj]: Should this be removed?

File: version-checks (hunk 1 above)
Comment (Therzok) [reply version-checks]: This is done to a branch.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.csproj (hunk 2 above)
Comment (KirillOsenkov) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.csproj]: This is what turns off using the checked in fork of VS Editor source.

**Pull request merged**